pantheon-bundle/frontend/src/app/search.tsx

```diff
@@ -52,9 +50,7 @@ class Search extends Component<IAppState, ISearchState> {
       alertOneVisible: true,
       checkNextPageRow: "",
       columns: ['Name', 'Description', 'Source Type', 'Source Name', 'Upload Time'],
-      confirmDelete: false,
       contentType: "module",
```

Comment:
Not a part of this ticket, but are we using it anywhere ?

Reply:
It's part of this ticket. when a delete action is taken, a confirmDelete staet is tracked.